test/runner.js

```diff
@@ -32,7 +30,7 @@ runtimeify({
   }
 
   const qemu = cp.spawn(qemuPath,
-    [ `${__dirname}/initrd`, '--verbose', '--nographic' ]
+    [ __dirname + '/initrd', '--verbose', '--nographic' ]
```

Comment:
No, im on Node 0.12, but I feel luke i'm missing out on ES6 without io.js, i'm going to install it and re-add ecerything the way it was before.